packages/devtools_app/NEXT_RELEASE_NOTES.md

```diff
@@ -1,38 +1,27 @@
 This is draft for future release notes, that are going to land on
 [the Flutter website](https://docs.flutter.dev/development/tools/devtools/release-notes).
 
-# DevTools 2.21.0 release notes
+# DevTools 2.22.0 release notes
```

Comment:
this shouldn't be changed to 2.22.0 - all these release notes are for 2.21.0 which has not been released yet, so just add your entry to the debugger section and leave all the rest

Reply:
Our update helper will reset this file anytime we do a major or minor version bump. We will probably need to undo the changes to this file so we don't remove the release notes